exp/capacity-planning.md

```diff
@@ -0,0 +1,96 @@
+
+<a name="gpu-slots"></a>
+### GPU slots
+
+An additional aspect to take into account when planning your resources is the number of required GPU slots (see [About GPU support](https://discourse.ubuntu.com/t/gpu-support/17768) for more information).
+
+GPUs have limited capacity that can be shared amongst multiple containers, and GPU slots are a way to fine-tune how many containers can run on a given node. In a cluster setup, you define the number of available GPU slots for each node (see [Configure GPU slots](tbd#configure-gpu-slots) for instructions).
+
+To determine the best number of GPU slots for a specific GPU model, consider the following aspects:
+
+- The memory that the GPU provides
+- The memory that a container uses
+- The number of parallel encoding pipelines that the GPU offers
+
+When you launch a container for an application, it reserves the number of GPU slots defined for the application on the node where it is launched. These GPU slots are marked as unavailable until the container is terminated. If no GPU slots are available on the node, containers that require a GPU will not be launched on it. Containers that don't require a GPU can still be launched.
+
+[note type="information" status="Important"]
+GPU slots are used to share GPUs amongst containers, but they do not impose limits on GPU usage. Therefore, increasing the number of required GPU slots for an application does not guarantee that more GPU resources are allocated to the corresponding application containers. For example, an intensive game that is configured to use one GPU slot might consume more GPU resources than a simple photo gallery app that is configured to use five GPU slots.
+
+The main purpose of GPU slots is to control the number of containers that are launched on a node, which reduces contention for GPU resources.
+[/note]
+
+<a name="overcommitting"></a>
+## Over-committing resources
+
+If the unused resources on a cluster node don't suffice to launch a container for an application with its defined resource requirements, the container cannot be launched. This behaviour is very restrictive, and in many cases unnecessary.
+
+Usually, a container doesn't use its dedicated CPU cores and memory at 100% all the time. Therefore, AMS allows over-committing available resources. By default, AMS uses a CPU allocation rate of `4` and a memory allocation rate of `2`, which means that it allows four times the number of vCPU cores and twice the amount of RAM per node. See [Configure allocation rates](tbd#configure-allocation-rates) for instructions on how to define the allocation rates for a node.
```

Comment:
We need to make the different between CPU and vCPU clear here. Physical CPU Cores * allocation rate = vCPUs